[INFO] Module binutils/2.38 loaded.
[INFO] Module zlib/1.2.13 loaded.
[INFO] Module GCCcore/.12.2.0 loaded.
[INFO] Module binutils/2.38 loaded.
[INFO] Module numactl/2.0.16 loaded.
[INFO] Module UCX/1.17.0 loaded.

Due to MODULEPATH changes, the following have been reloaded:
  1) UCX/1.17.0     2) binutils/2.38     3) numactl/2.0.16     4) zlib/1.2.13

The following have been reloaded with a version change:
  1) GCCcore/.11.3.0 => GCCcore/.12.2.0

Gurobi shell based on Python 3.10.4 can be launched with command `gurobi.sh`
Gurobi Python Interface can be loaded in Python 3.10.4 with 'import gurobipy'
[INFO] Module Gurobi/10.0.0 loaded.

The following have been reloaded with a version change:
  1) zlib/1.2.13 => zlib/1.2.12

[INFO] Module Python/3.10.4 loaded.
Traceback (most recent call last):
  File "/cvmfs/software.hpc.rwth.de/Linux/RH8/x86_64/intel/sapphirerapids/software/Python/3.10.4-GCCcore-12.2.0/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/cvmfs/software.hpc.rwth.de/Linux/RH8/x86_64/intel/sapphirerapids/software/Python/3.10.4-GCCcore-12.2.0/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/rwthfs/rz/cluster/home/wx350715/Kobra/rehabiliation-admission-scheduling/src/__main__.py", line 185, in <module>
    main()
  File "/rwthfs/rz/cluster/home/wx350715/Kobra/rehabiliation-admission-scheduling/src/__main__.py", line 142, in main
    solution = solver.solve_model()
  File "/rwthfs/rz/cluster/home/wx350715/Kobra/rehabiliation-admission-scheduling/src/solvers/solver.py", line 66, in solve_model
    solution = self._solve_model()
  File "/rwthfs/rz/cluster/home/wx350715/Kobra/rehabiliation-admission-scheduling/src/solvers/benders.py", line 185, in _solve_model
    solution = self._extract_solution()
  File "/rwthfs/rz/cluster/home/wx350715/Kobra/rehabiliation-admission-scheduling/src/solvers/benders.py", line 780, in _extract_solution
    day_solution = self.subsolver.get_day_solution(d, patients[d])
  File "/rwthfs/rz/cluster/home/wx350715/Kobra/rehabiliation-admission-scheduling/src/solvers/subsolvers/subsolver.py", line 181, in get_day_solution
    return self._get_day_solution(day, patients)
  File "/rwthfs/rz/cluster/home/wx350715/Kobra/rehabiliation-admission-scheduling/src/solvers/subsolvers/cp_subsolver.py", line 283, in _get_day_solution
    assert status == cp_model.OPTIMAL, f"Subsolver on day {day} is not optimal"
AssertionError: Subsolver on day 10 is not optimal
